source/reference/deprecated/mc-admin-top.rst

```diff
@@ -10,11 +10,9 @@
 
 .. mc:: mc admin top
 
-.. note::
+.. versionchanged:: RELEASE.2022-08-11T00-30-48Z
```

Comment:
This one is a direct move of something already deprecated. Could start making redirects, there's the Sphinx extension Ravind mentioned.

Reply:
The easiest path is something like 

```

.. rawhtml:: 
   
   <meta http-equiv="refresh" content = "0; url=REDIRURL" />

```

It's not as nice for SEO compared to a 301 from nginx, but we don't have a good way of managing those rules, so that might be an option.

Reply:
@ravindk89 Would that go in a stub page at the original location?

Reply:
`sphinx-reredirects` is being fussy, I'll dig into that in a separate PR.